src/features/partners/PartnerStats.jsx

```diff
@@ -8,7 +8,7 @@ import MemberEnrollmentList from '../enrollments/MemberEnrollmentList';
 import MembershipProvider from '../members/MembershipProvider';
 import usePartner from './usePartner';
 import StatCard from './StatCard';
-import ManagementMenu from './ManagementMenu';
+import ManagementToolbar from './ManagementToolbar';
```

Comment:
Has the `ManagementMenu` been entirely removed from every page? If so we should probably remove the component file too.

Reply:
File has been removed